ignite/cmd/chain_build.go

```diff
@@ -88,10 +89,10 @@ func chainBuildHandler(cmd *cobra.Command, args []string) error {
 	}
 
 	if output == "" {
-		fmt.Printf("🗃  Installed. Use with: %s\n", infoColor(binaryName))
+		fmt.Printf("🗃  Installed. Use with: %s\n", colors.Info(binaryName))
```

Comment:
We also have other colors in the cmd package that can be moved to the `colors` pkg, can you please create an issue so we can come back to it later in another PR?

Reply:
https://github.com/ignite-hq/cli/issues/2432